lib/screens/home/home.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:brew_crew/screens/home/settings_form.dart';
+import 'package:brew_crew/services/auth.dart';
+import 'package:flutter/material.dart';
+import 'package:brew_crew/services/database.dart';
+import 'package:provider/provider.dart';
+import 'package:brew_crew/screens/home/brew_list.dart';
+import 'package:brew_crew/models/brew.dart';
+
+class Home extends StatelessWidget {
+  final AuthService _auth = AuthService();
+
+  @override
+  Widget build(BuildContext context) {
+    void _showSettingsPanel() {
+      showModalBottomSheet(
+          context: context,
```

Comment:
nie wywala ci błędu ta funkcja w buildzie?

Reply:
to jest poprawne językowo, ale to niekoniecznie dobra praktyka. takie wewnętrzne funkcje mają sens zazwyczaj tylko w ciele hooków

Reply:
> nie wywala ci błędu ta funkcja w buildzie?

nie

Reply:
tutaj to zrobiłem żeby przekazać context,co jest obowiązkowym argumentem

Reply:
to zrób osobną funkcję i przekaż do niej BuildContext